packages/dm-core-plugins/src/job/JobControl.tsx

```diff
@@ -75,8 +77,19 @@ export const JobControl = (props: IUIPlugin) => {
     if (!jobEntity) return
     jobEntity.triggeredBy =
       tokenData?.preferred_username || tokenData?.name || 'Anonymous'
-    // @ts-ignore
-    if (asCronJob) jobEntity.schedule = schedule
+    if (asCronJob) {
+      // @ts-ignore
```

Comment:
I agree with linting here - why are we updating state in a non-immutable way like this? Goes for both triggeredBy and schedule

Reply:
The linter did not complain, its typescript that is not able to see that the job is a recurring job.
However, no reason to mutate a state object neither :slightly_smiling_face: